server/src/main/scala/controllers/Application.scala

```diff
@@ -126,24 +126,35 @@ trait UserRoleProviderLike {
   def getRoles(config: Configuration, headers: Headers, session: Session): Set[Role]
 
   def getLoggedInUser(config: Configuration, headers: Headers, session: Session): LoggedInUser = {
-    val baseRoles =  Set()
+    val baseRoles = Set()
     val roles: Set[Role] =
       getRoles(config, headers, session) ++ baseRoles
     LoggedInUser(
       userName = headers.get("X-Auth-Username").getOrElse("Unknown"),
       id = headers.get("X-Auth-Userid").getOrElse("Unknown"),
       email = headers.get("X-Auth-Email").getOrElse("Unknown"),
       roles = roles
-    )
+      )
   }
 }
 
+object DrtActorSystem extends AirportConfProvider {
+  implicit val actorSystem: ActorSystem = ActorSystem("DRT")
+  implicit val mat: ActorMaterializer = ActorMaterializer()
+  implicit val ec: ExecutionContextExecutor = ExecutionContext.global
+  val config: Configuration = new Configuration(ConfigFactory.load)
+
+  val drtSystem: DrtSystemInterface =
+    if (isTestEnvironment) drtTestSystem
+    else DrtSystem(config, getPortConfFromEnvVar)
+
+  lazy val drtTestSystem: TestDrtSystem = TestDrtSystem(config, getPortConfFromEnvVar)
```

Comment:
should we make the production system into a lazy val first as well - to make them the same? or inline the TestSystem - would be happy with either.

Reply:
Ok I've made them both lazy vals